test/test_front_matter_defaults.rb

```diff
@@ -27,6 +27,30 @@ class TestFrontMatterDefaults < JekyllUnitTest
     end
   end
 
+  context "A site with full front matter defaults (glob)" do
+    setup do
+      @site = fixture_site({
+        "defaults" => [{
+          "scope"  => {
+            "path" => "contacts/*.html",
```

Comment:
Since we're using `Dir.glob`, according to [docs](https://ruby-doc.org/core-2.2.0/Dir.html#method-c-glob), `*.html` will match only files on top directory, to match _any_ HTML file, a `**/*.html` pattern should we used